pkg/benchmarks/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "peggy-benchmarks",
+  "version": "1.0.0",
+  "description": "Peggy Benchmark Suite",
+  "private": true,
+  "scripts": {
+    "test": "./run",
+    "build": "webpack",
+    "postbuild": "../../tools/header ../../docs/js/benchmark-bundle.min.js"
+  },
+  "keywords": [],
+  "author": "Joe Hildebrand <joe-github@cursive.net>",
+  "license": "MIT",
+  "dependencies": {
+    "peggy": "*"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.15",
+    "@babel/preset-env": "^7.13.15",
+    "babel-loader": "^8.2.2",
+    "webpack": "^5.33.2",
+    "webpack-cli": "^4.6.0"
+  }
+}
```

Comment:
putting secondary packages into the main repo to create a monorepo is a classic security and maintainability disaster

there's literally never a good reason to do this.  this means that the subpackages aren't real packages anymore. 

want just one package?  nope, gotta download all of them.

want the simplicity of one piece?  nope, gotta understand all of them.

want standard tooling you're allowed to use in any corporate context?  nope, can't do this on your work pc at all.

literally no value is delivered here of any kind.  all this means is that the repo has a secondary alternate model that only a few people understand, and in order to contribute to peg at all, first you have to learn a completely different package management model, and sort out how the bizarro-land rube goldberg build system actually works

When futagoza did this three years ago, contributions hard-stopped immediately

please don't do this `:(`  

it is critical and important for public open source projects to work the regular way

this is a death knell

even high touch monorepos eventually get turned back normal because the dip in usage is too bad

this library is already almost dead and has already been dragged through three other unrelated monorepo concepts.  everyone's sick of it and a lot of old developers have already left in disgust, because the people taking over are wasting our time forcing us to learn their hipster tools

Reply:
i cannot point to a single monorepo across all of NPM that was popular and didn't immediately and permanently lose 50% of all its traffic

i tried to tell you "lerna is a nightmare" but you said "don't worry, i'm not using alternate package managers."

yes, you are.  

[i see you locked me out of my own issue just now](https://github.com/peggyjs/peggy/issues/76).

seems like you're being pretty explicit that even politely stated alternative ideas aren't welcome

Reply:
oh, nevermind, on reload there's a link to some new system called a discussion

i guess it doesn't put those there on hot-act like it does closures.  weird

Reply:
I really don't understand your corporate context argument.  Can you say more about that please?  Assume in your answer that you'll be able to contribute just fine with npm once I remove the pnpm dependency.

Reply:
> I really don't understand your corporate context argument. Can you say more about that please? Assume in your answer that you'll be able to contribute just fine with npm once I remove the pnpm dependency.

Many can't install alternate package managers because corporate hasn't verified whether they do what they say on the box, and isn't willing to add them to the watchlist

Package managers have access to protected locations on disk that can run binaries.  They're higher stakes than typical JS libs.  Typical JS libs, you can just scan them to see if they have any on-install or on-build hooks then call it a day

Reply:
in this scenario `pnpm` is installed via `npm`, it's not a global install like `yarn` would be. I think this should make it more palatable?

Reply:
> in this scenario pnpm is installed via npm, it's not a global install like yarn would be. I think this should make it more palatable?

Until the security team looks it over, which will probably be never, unfortunately no.

There's a cost to verifying whether a tool is still safe, on each release. It's hard to get an employer to commit to adding a name to that list.

Reply:
> if we move forward I'll remove pnpm as a dependency and from being called in the scripts.

awesome, thank you

sorry to be weird about it